src/api/factories/ListingItemFactory.ts

```diff
@@ -21,33 +22,33 @@ export class ListingItemFactory {
      * @param ListingItemMessage
      * @param marketId
      *
-     * @returns {ListingItemMessage}
+     * @returns {ListingItemTemplatePostRequest}
      */
 
-    public async getMessage(data: ListingItemMessage, marketId: number): Promise<ListingItemMessage> {
+    public async getMessage(data: ListingItemTemplatePostRequest, marketId: number): Promise<ListingItemMessage> {
         return {
             hash: data.hash,
-            listingItemTemplateId: data.listingItemTemplateId,
+            listingItemTemplateId: data.id,
             marketId,
             information: {
-                title: data.information.title,
-                shortDescription: data.information.shortDescription,
-                longDescription: data.information.longDescription,
+                title: data.ItemInformation.title,
+                shortDescription: data.ItemInformation.shortDescription,
+                longDescription: data.ItemInformation.longDescription,
                 itemCategory: {
```

Comment:
if data contains ItemInformation instead of itemInformation, you're using the model, which is wrong